x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlIndexRollover.java

```diff
@@ -60,7 +59,7 @@ public MlIndexRollover(List<IndexPatternAndAlias> indicesToRollover, IndexNameEx
     public boolean isMinTransportVersionSupported(TransportVersion minTransportVersion) {
         // Wait for all nodes to be upgraded to ensure that the
         // newly created index will be of the latest version.
-        return minTransportVersion.supports(TransportVersions.V_8_18_0);
+        return true;
```

Comment:
Presumably `minTransportVersion` will always be compatible? I assume this is only used in the scenario of a rolling upgrade?